apps/src/netsim/NetSimNodeRouter.js

```diff
@@ -110,16 +126,16 @@ var RouterStatus = NetSimNodeRouter.RouterStatus;
 /**
  * Updates router status and lastPing time in lobby table - both keepAlive
  * and making sure router's connection count is valid.
- * @param onComplete
+ * @param {function} [onComplete] - Optional success/failure callback
```

Comment:
Square brackets denote an [optional parameter in JSDoc](http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values).